tests/test_qt_js_integration.py

```diff
@@ -72,8 +72,8 @@ def check_call(expected):
     with events.delaying_signals():
         events.cells_added_slot(["2"])
 
-        with pytest.raises(pytestqt.exceptions.TimeoutError):
-            expected = [call(["1"]), call(["2"])]
+        expected = [call(["1"]), call(["2"])]
+        with pytest.raises((AssertionError, pytestqt_TimeoutError)):
```

Comment:
In some versions pytestqt's `qtbot.waitUntil` will fail with `AssertionError` and others with `pytestqt.exceptions.TimeoutError`.

Reply:
Yeah it was a [breaking change in 4.0.0](https://github.com/pytest-dev/pytest-qt/blob/master/CHANGELOG.rst#400-2021-06-03).

Reply:
> waitUntil now raises a TimeoutError when a timeout occurs to make the cause of the timeout more explicit (https://github.com/pytest-dev/pytest-qt/pull/222).

Reply:
PRs are welcome! 😉